subsys/bluetooth/mesh/pb_gatt.c

```diff
@@ -246,6 +251,7 @@ const struct prov_bearer bt_mesh_pb_gatt = {
 #endif
 #if defined(CONFIG_BT_MESH_PB_GATT)
```

Comment:
I guess this is useless conditional compilation since CONFIG_BT_MESH_PB_GATT is always `y` for this file, but it is out of scope of your changes.